### ISSUE TITLE
subprocess call fails when issued from Robot Framework

### ISSUE THREAD

**Issue by mwidera** [beelit94/python-terraform#73]
I've noticed following behavior:

Executing via `python script.py`:
```tf = Terraform(workdir='".")
tf.init(input=False, capture_output=True)
tf.init(input=False, capture_output=False)
```
Works fine, but when command is issued via robot framework subprocess starts to yield an error:
`robot testscenario.robot`

```
UnsupportedOperation: fileno
Traceback (most recent call last):
  File ".../lib/python3.7/site-packages/python_terraform/__init__.py", line 296, in cmd
    cwd=working_folder, env=environ_vars)
  File "/usr/lib/python3.7/subprocess.py", line 753, in __init__
    errread, errwrite) = self._get_handles(stdin, stdout, stderr)
  File "/usr/lib/python3.7/subprocess.py", line 1388, in _get_handles
    c2pwrite = stdout.fileno()
```

To fix this problem easiest thing to do is to set both stdout and stderr in cmd method to None and this might be done via "capture_output" flag which could be set to certain string to work with this corner case

**Comment by mwidera** [comment]
Related PR: https://github.com/beelit94/python-terraform/pull/74

**Comment by Spikeophant** [comment]
This PR was merged to Develop, will be in the 0.10.2 release that is going to be out in a couple weeks, I appreciate the PR.
